src/lib.ts

```diff
@@ -499,11 +499,9 @@ export function getBanishedMonsters(): Map<Item | Skill, Monster> {
         null,
       ].includes(banisherItem)
     ) {
-      if (Skill.get(banisher) === $skill.none) {
-        break;
-      } else {
-        result.set(Skill.get(banisher), Monster.get(foe));
-      }
+      const skill = $skill.get(banisher);
+      if (!skill) break;
```

Comment:
```suggestion
      if (!skill) continue;
```

Reply:
Yeah fair enough. This is a behavioural change but is desired probably. I think it was originally break to deal with an indivisible-by-3 number of elements or something